### PR TITLE
Add compounds to vapor pressure batch8

### PR DESCRIPTION
**Pull request by sumitnegi7**

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -2148,6 +2148,606 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    minimumTemperature: 1.76,
+    maximumTemperature: 5.2,
+    pressureAtMinimumTemperature: 1.46e3,
+    pressureAtMaximumTemperature: 2.284e5,
+  },
+
+  Heptadecane: {
+    name: 'Heptadecane',
+    formula: 'C17H36',
+    CAS: '629-79-7',
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2365]: 629-78-7

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -2148,6 +2148,606 @@ export const DIPPRVaporPressureDictionary: Record<
...
+    formula: 'C7H14',
+    CAS: '592-76-7',
+    C1: 65.922,
+    C2: -6189,
+    C3: -6.3629,
+    C4: 2.01e-17,
+    C5: 6,
+    minimumTemperature: 154.12,
+    maximumTemperature: 537.4,
+    pressureAtMinimumTemperature: 1.84e-3,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:2493]: 1.86E-03

**Review by saikat-samanta — APPROVED** [review]
Rest good 👍

**Review by qur786 — COMMENTED** [review]
All Good

File: src/data/vapor-pressure-constants.ts (hunk 1 above)
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:2365]:
```suggestion
    CAS: '629-78-7',
```

File: src/data/vapor-pressure-constants.ts (hunk 2 above)
Comment (maneetgoyal) [inline src/data/vapor-pressure-constants.ts:2493]:
```suggestion
    pressureAtMinimumTemperature: 1.86e-3,
```

**Review by maneetgoyal — APPROVED** [review]
Minor errors

**Pull request merged**